superduperdb/base/build.py

```diff
@@ -117,6 +117,18 @@ def _build_databackend_impl(uri, mapping, type: str = 'data_backend'):
         name = uri.split('/')[-1]
         conn = mongomock.MongoClient()
         return mapping['mongodb'](conn, name)
+    
+    elif uri.startswith('postgres://') or uri.startswith("postgresql://"):
+        name = uri.split('//')[0]
+        if type == 'data_backend':
+            ibis_conn = ibis.connect(uri)
+            return mapping['ibis'](ibis_conn, name)
+        else:
+            assert type == 'metadata'
+            from sqlalchemy import create_engine
+
+            sql_conn = create_engine(uri)
+            return mapping['sqlalchemy'](sql_conn, name)
```

Comment:
Please remove this, we can directly load the URI with ibis

Reply:
I will remove this